docs/building-experiments/metrics/creating-metrics.md

```diff
@@ -34,14 +34,48 @@ You may actually want to create a metric that is a ratio of two facts. To do thi
 
 In the example above, we're creating a metric that corresponds to the revenue per user per purchase.
 
-5. (Optional) Select a filter
+5. Select a minimum detectable effect
 
-When you created an assignment SQL above, you may have also created additional dimensions, i.e. country or browser. These dimensions are now available under the **Definitions** > **Dimensions SQL**.
+The minimum detectable effect refers to the smallest effect you want to reliably detect in experiments. The lower the minimum detectable effect you set, the more samples you'll need for the experiment to reach conclusive results.
 
-![Select filter](../../../static/img/building-experiments/filter-on-dimensions-create-metric.gif)
+## Metric aggregation types
+
+### Sum
+
+Sum computes metrics that are typically interpreted as averages per entity. If the fact value is NULL, it is imputed as 0.
+
+$\frac{\text{SUM of fact value}}{\text{Number of unique entities}}$
+
+Examples: average revenue per user, sign-up rate, minutes streamed per user, average order value.
+
+### Count Distinct
+
+Count Distinct leverages SQL's ``COUNT DISTINCT`` to compute a unique average per entity. If the fact value is NULL, it is discarded.
+
+$\frac{\text{Sum of (COUNT DISTINCT of fact value for each unique entity)}}{\text{Number of unique entities}}$
+
+Examples: unique videos watched per user, unique articles viewed per visitor, unique products viewed per user.
+
+### Count
+
+Count leverages SQL's ``COUNT`` to compute a total count of events per entity. If the fact value is NULL, it is discarded.
+
+$\frac{\text{Sum of (COUNT of fact value for each unique entity)}}{\text{Number of unique entities}}$
+
+Examples: videos watched per user, articles viewed per visitor, orders per user.
+
+### Retention
+
+Retention metrics measure the proportion of entities who have at least one fact value appear after a fixed number of days (X) from experiment assignment. For example, a 7-day retention metric might measure the proportion of users who re-visit a website at least 7 days after being assigned to the experiment. 
+
+$\frac{\text{Sum of \{1 if fact value is non-null after X days of assignment, else 0, for each unique entity\}}}{\text{Number of unique entities}}$
+
+
+### Conversion
+
+Conversion metrics measure the proportion of entities who have at least one fact value appear within a fixed number of days (X) from experiment assignment. For example, a 7-day conversion metric might measure the proportion of users who sign up for a free trial within 7 days of being assigned to the experiment. 
+
+$\frac{\text{Sum of \{1 if fact value is non-null within X days of assignment, else 0, for each unique entity\}}}{\text{Number of unique entities}}$
```

Comment:
```suggestion
$\frac{\text{Sum of \{1 if fact value is non-null within X days of assignment, else 0, for each unique entity\}}}{\text{Number of assignments}}$
```

Maybe assignments is better than entities here? (entity is "User" vs "Visitor"). But this is correct. For both retention and conversion we do count all assignment, even if they haven't been in assigned for the entire period (e.g. assigned 3 days ago with a 7 day window)

Reply:
Going to change this to "unique entities assigned" to be super super clear